packages/typespec-vscode/package.json

```diff
@@ -273,13 +273,18 @@
     "test:extension": "vitest run --root test/extension"
   },
   "devDependencies": {
+    "@nut-tree-fork/nut-js": "^4.2.6",
     "@types/cross-spawn": "~6.0.6",
     "@types/mocha": "^10.0.9",
     "@types/node": "~24.0.3",
     "@types/semver": "^7.5.8",
     "@types/vscode": "~1.100.0",
     "@types/which": "^3.0.4",
     "@typespec/compiler": "workspace:^",
+    "@typespec/http": "workspace:^",
+    "@typespec/openapi": "workspace:^",
+    "@typespec/openapi3": "workspace:^",
+    "@typespec/http-client-js": "workspace:^",
```

Comment:
leaving same comments here too but we should not take those extra dependencnies here. the UI test should be as basic as possible and mock those things if needed. 
it not the goal of the ui to test the compiler and the libraries.

Reply:
Updated. Have removed these dependencies.

Reply:
Updated. These packages have been removed from `package.json`.